### DISCUSSION TITLE
Hue Dimmer Switch - Unable to connect

### DISCUSSION THREAD

**Discussion by TvojTatk0** [Koenkk/zigbee2mqtt#13359] [Devices]
Hello everyone.
Sorry for this rookie question but I tried google/github/reddt and didn't find answer for my problem.
I succesfuly added 3 Philips hue celeing lights to my home assistant using Zigbee2mqtt and Mosquitto broker. However I'm unable to add Philips Hue Dimmer Switch (latest model) to my network (none of the 3).  Problem is that I just don't see them in Z2M dashboard as a new device. I reset them holding all  4 buttons at the same time, then green/red LED flickering occured and nothing else happend.  What I'm doing wrong?
I canno't see anything else in this manual https://www.zigbee2mqtt.io/devices/929002398602.html

Setup:
Rasspberry 3b+
Home assistant v 2022.7.7
Mosquitto broker v 6.1.2
Zigbee2MQTT v1.26.0-1
Zigbee CC2531 USB stick
Hue dimmer switch:  https://www.zigbee2mqtt.io/devices/929002398602.html
Hue lights: https://www.zigbee2mqtt.io/devices/3261330P6.html

Thanks!

**Comment by gerard33** [comment]
Did you perform this step first?
`Factory reset the Hue dimmer switch by pressing and holding the setup button on the back for 10 seconds.`

Using that I was able to add my Hue dimmer (v1) to Z2M without any problems and without the need to hold all 4 buttons at the same time.

**Reply by TvojTatk0** [reply]
I didn't knew that this v2 version also have physical reset/setup button BUT under battery cover. After holding it for 10 seconds I was able to add switch into my network.
Thanks for your advice, problem solved :)
